include/envoy/ratelimit/BUILD

```diff
@@ -13,6 +13,9 @@ envoy_cc_library(
     hdrs = ["ratelimit.h"],
     external_deps = ["abseil_optional"],
     deps = [
+        "//include/envoy/registry",
```

Comment:
This header file should be moved entirely into extensions

Reply:
I could not completely move this to extensions  because, the router implementation https://github.com/envoyproxy/envoy/blob/master/source/common/router/router_ratelimit.cc uses `Descriptor` and `DescriptorEntry`  classes.  So I kept those classes here.